core/arch/arm/plat-imx/drivers/tzc380.c

```diff
@@ -21,8 +22,30 @@
  */
 #ifndef TZASC2_BASE
 #define TZASC2_BASE			0
+#else
+register_phys_mem(MEM_AREA_IO_SEC, TZASC2_BASE, TZASC_SIZE);
 #endif
 
+register_phys_mem(MEM_AREA_IO_SEC, TZASC_BASE, TZASC_SIZE);
```

Comment:
How could this driver work without this? It doesn't seem platform-specific code was registering the memory either?
Should this be a separate commit?

Reply:
The TZASC registers are registered by https://github.com/OP-TEE/optee_os/blob/master/core/arch/arm/plat-imx/main.c#L64
`AIPSx_BASE` and `AIPSx_SIZE` cover multiple peripheral/IP registers. That's how the TZASC registers were registered.

Reply:
Got it. So wouldn't it make sense to have one commit just for these two `register_phys_mem()`?

Anyways, for all commits:

`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

Reply:
Ok, no problem!